### DISCUSSION TITLE
Feature Request - Flutter Firebase Messaging for a non default app

### DISCUSSION THREAD

**Discussion by ma-altaiir** [firebase/flutterfire#5565] [Feature request]
I am new to flutter and developement in general, I have been looking arround firebase plugins for flutter and noticed many of them can actually use non default firebase app , i tried to make my code receive notifications from two different projects but failed, how can one register a device with two different firebase apps ? Is there any http endpoint to register device by providing senderId and scope? Will be there any support for an instanceFor(FirebaseApp app) for messaging plugin ?

**Comment by DRSchlaubi** [comment]
Why was the issue for this closed?
There is code for this method here: https://github.com/FirebaseExtended/flutterfire/blob/1db77b689c1fa856194c628d8e11e69f848f82e0/packages/firebase_messaging/firebase_messaging/lib/src/messaging.dart#L32-L52
But it's commented out

**Reply by fullflash** [reply]
seems the native part of this plugin just requires google service info plist config and has no support for inapp config.
